test/parallel/test-fs-readfile-empty.js

```diff
@@ -27,20 +27,21 @@ require('../common');
 const assert = require('assert');
 const fs = require('fs');
 const fixtures = require('../common/fixtures');
+const { mustCall } = require('../common')
 
 const fn = fixtures.path('empty.txt');
 
-fs.readFile(fn, function(err, data) {
+fs.readFile(fn, mustCall(function (err, data) {
```

Comment:
please remove the space after `function`:
`function (err, data)` -> `function(err, data)`